web/static/css/_global.scss

```diff
@@ -1626,13 +1626,28 @@ a.disabled {
     right: 0;
     width: 400px;
   }
+  &.single-date-picker {
+    top: -65px;
+    i[disabled] {
+      @extend .grey-text;
+    }
+  }
 }
 
 .Cal__Header__root {
   display:none;
 }
 
-.Cal__Day__selection:before {
+.row.start-date {
```

Comment:
(change) 

If the only custom style needed for the start-date row is the margin-bottom, better use `.row.mb0` or something alike. When end date is added, is going to be in this same row so the start-date name is not appropriate.